llvm/test/CMakeLists.txt

```diff
@@ -174,6 +174,10 @@ if(LLVM_TOOL_REPO_FRAGMENTS_BUILD)
   set(LLVM_TEST_DEPENDS ${LLVM_TEST_DEPENDS} repo-fragments)
 endif()
 
+if(LLVM_TOOL_REPO_CREATE_TICKET)
```

Comment:
Other advanced property for variables have `_BUILD` in the end of variable names.  Should it be `if(LLVM_TOOL_REPO_CREATE_TICKET_BUILD)`?

Reply:
Typo fixed by commit 0fc2cdcb404f16a21e9ab68467607c296f5ea047 after which the commit checks now pass.